test/built-ins/BigInt/prototype/valueOf/this-value.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-bigint.prototype.valueof
+description: BigInt.prototype.valueOf this value coercion
+info: >
+  BigInt.prototype.valueOf ( )
+
+  Return ? thisBigIntValue(this value). 
+includes: [typeCoercion.js]
+features: [BigInt, arrow-function, Symbol, Symbol.toPrimitive]
+---*/
+
+testCoercibleToBigIntThisValue(
+  0n,
+  (x) => assert.sameValue(0n, BigInt.prototype.valueOf.call(x))
+);
```

Comment:
static direct calls would provide a better reading for this. Expanding the text for readability can help here.